repos/system_upgrade/el7toel8/models/repositoriesfacts.py

```diff
@@ -25,3 +25,11 @@ class RepositoriesFacts(Model):
     topic = SystemFactsTopic
 
     repositories = fields.List(fields.Model(RepositoryFile))
+
```

Comment:
add the deprecation and suprress deprecations decorators to make it more clear that this is deprecated to be used by anyone else.

Reply:
github don't like forced pushes similar to me. Moving your comment to a right place